packages/components/src/guide/stories/index.js

```diff
@@ -39,40 +33,19 @@ const ModalExample = ( { numberOfPages, ...props } ) => {
 			{ isOpen && (
 				<Guide
 					{ ...props }
-					onFinish={ closeGuide }
-					pages={ Array.from( { length: numberOfPages } ).map(
-						( _, page ) => ( {
-							content: (
-								<>
-									<h1>
-										Page { page + 1 } of { numberOfPages }
-									</h1>
-									<p>{ loremIpsum }</p>
-								</>
-							),
-						} )
-					) }
+					onFinish={ ( ...finishArgs ) => {
+						closeGuide( ...finishArgs );
+						onFinish( ...finishArgs );
+					} }
 				/>
 			) }
 		</>
 	);
 };
 
-export const _default = () => {
-	const finishButtonText = text( 'finishButtonText', 'Finish' );
-	const contentLabel = text( 'title', 'Guide title' );
-	const numberOfPages = number( 'numberOfPages', 3, {
```

Comment:
`numberOfPages` is not an actual component prop. It was being used as a Storybook-only utility prop to generate the `pages` array.